udp-ballerina/tests/mock_servers.bal

```diff
@@ -38,7 +38,7 @@ service on logServer {
 
 service on echoServer {
 
-    remote function onBytes(readonly & byte[] data, Caller caller) returns (readonly & byte[])|Error? {
+    remote function onBytes(Caller caller, readonly & byte[] data) returns (readonly & byte[])|Error? {
```

Comment:
In this case do we need the caller ?

Reply:
Caller can be removed here

Reply:
Updated

Reply:
Do we have test using caller as well ?

Reply:
yes, in botServer service onDatagram() method line no. 58